typescript/servicecatalog/portfolio-with-ec2-product/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "product-stack",
+  "version": "0.1.0",
+  "bin": {
+    "product-stack": "bin/ec2-product-stack.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest",
+    "cdk": "cdk"
+  },
+  "devDependencies": {
+    "@aws-cdk/assertions": "1.134.0",
```

Comment:
```suggestion
```
You shouldn't need this since you're using v2. You'll just use `aws-cdk-lib/assertions`

Reply:
Ack, and changed in latest commit.